Bugzilla/Install.pm

```diff
@@ -500,9 +500,14 @@ sub _prompt_for_password {
         print "\n", get_text('install_confirm_password'), ' ';
         my $pass2 = <STDIN>;
         chomp $pass2;
-        eval { validate_password($password, $pass2); };
-        if ($@) {
-            print "\n$@\n";
+        my $pwqc = Bugzilla->passwdqc;
+        my $ok = $pwqc->validate_password($password);
+        if (!$ok) {
+            print "\n", $pwqc->reason, "\n";
+            undef $password;
```

Comment:
it may be worth documenting the default password complexity requirements in the README, to ease the initial setup.